dockstore-webservice/src/test/java/io/dockstore/webservice/languages/NextflowHandlerTest.java

```diff
@@ -0,0 +1,19 @@
+package io.dockstore.webservice.languages;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class NextflowHandlerTest {
+
+    @Test
+    public void testGetRelativeImportPathFromLine() {
```

Comment:
A test that a commented out line doesn't return an import would be a bonus, e.g.,

```
#include { RNASEQ } from './modules/rnaseq'", "/main.nf
```
(or whatever the syntax for a comment in NF is).